<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>2125</width>
    <height>1076</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>ÎÇòÏä§Îã•100 Ìò∏Í∞ÄÏ∞Ω ÏòàÏ†ú (ÌôïÏû•Ìòï)</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="mainLayout">
    <item>
     <widget class="QFrame" name="frame_left">
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_left">
       <item>
        <widget class="QTableWidget" name="tableWidget">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <column>
          <property name="text">
           <string>New Column</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>New Column</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>New Column</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>New Column</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>New Column</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>New Column</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>New Column</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>New Column</string>
          </property>
         </column>
        </widget>
       </item>
       <item>
        <widget class="QFrame" name="frame_top">
         <layout class="QHBoxLayout" name="horizontalLayout_top">
          <item>
           <widget class="QLabel" name="label_market">
            <property name="text">
             <string>ÎÇòÏä§Îã•100</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="label_price">
            <property name="text">
             <string>24,590.50</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="label_change">
            <property name="text">
             <string>‚ñ≤ 38.75  -0.16%</string>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="horizontalSpacer">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>0</width>
              <height>0</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QPushButton" name="button_chart">
            <property name="text">
             <string>Ï∞®Ìä∏</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="button_account">
            <property name="text">
             <string>Í≥ÑÏ¢åÍ¥ÄÎ¶¨</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="button_setting">
            <property name="text">
             <string>ÌôòÍ≤ΩÏÑ§Ï†ï</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_stocklist">
         <property name="text">
          <string>Ï¢ÖÎ™©Î™©Î°ù</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QTableWidget" name="table_hoga">
         <column>
          <property name="text">
           <string>MIT</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Îß§ÎèÑ</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Í±¥Ïàò</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>ÏûîÎüâ</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Í≥†Ï†ï</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>ÏûîÎüâ</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Í±¥Ïàò</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Îß§Ïàò</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>MIT</string>
          </property>
         </column>
        </widget>
       </item>
       <item>
        <widget class="QFrame" name="frame_buttons">
         <layout class="QHBoxLayout" name="horizontalLayout_buttons">
          <item>
           <widget class="QPushButton" name="button_sell_res_cancel">
            <property name="text">
             <string>Îß§ÎèÑÏòàÏïΩÏ∑®ÏÜå</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="button_sell_cancel">
            <property name="text">
             <string>Îß§ÎèÑÏ∑®ÏÜå</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="button_all_cancel">
            <property name="text">
             <string>ÏùºÍ¥ÑÏ∑®ÏÜå</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="button_item_cancel">
            <property name="text">
             <string>Ï¢ÖÎ™©Ï∑®ÏÜå</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="button_buy_cancel">
            <property name="text">
             <string>Îß§ÏàòÏ∑®ÏÜå</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="button_buy_res_cancel">
            <property name="text">
             <string>Îß§ÏàòÏòàÏïΩÏ∑®ÏÜå</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QTabWidget" name="tabWidget">
         <property name="currentIndex">
          <number>0</number>
         </property>
         <widget class="QWidget" name="tab">
          <attribute name="title">
           <string>Ï†ÑÏ≤¥</string>
          </attribute>
          <layout class="QHBoxLayout" name="horizontalLayout_3">
           <item>
            <widget class="QTableWidget" name="tableWidget_6">
             <column>
              <property name="text">
               <string>New Column</string>
              </property>
             </column>
             <column>
              <property name="text">
               <string>New Column</string>
              </property>
             </column>
             <column>
              <property name="text">
               <string>New Column</string>
              </property>
             </column>
             <column>
              <property name="text">
               <string>New Column</string>
              </property>
             </column>
             <column>
              <property name="text">
               <string>New Column</string>
              </property>
             </column>
             <column>
              <property name="text">
               <string>New Column</string>
              </property>
             </column>
             <column>
              <property name="text">
               <string>New Column</string>
              </property>
             </column>
            </widget>
           </item>
          </layout>
         </widget>
         <widget class="QWidget" name="tab_2">
          <attribute name="title">
           <string>ÏûîÍ≥†</string>
          </attribute>
          <layout class="QGridLayout" name="gridLayout_2">
           <item row="0" column="0">
            <widget class="QTableWidget" name="tableWidget">
             <column>
              <property name="text">
               <string>New Column</string>
              </property>
             </column>
             <column>
              <property name="text">
               <string>New Column</string>
              </property>
             </column>
             <column>
              <property name="text">
               <string>New Column</string>
              </property>
             </column>
             <column>
              <property name="text">
               <string>New Column</string>
              </property>
             </column>
            </widget>
           </item>
          </layout>
         </widget>
         <widget class="QWidget" name="tab_3">
          <attribute name="title">
           <string>ÎØ∏Ï≤¥Í≤∞</string>
          </attribute>
          <layout class="QHBoxLayout" name="horizontalLayout_2">
           <item>
            <widget class="QTableWidget" name="tableWidget_5">
             <column>
              <property name="text">
               <string>New Column</string>
              </property>
             </column>
             <column>
              <property name="text">
               <string>New Column</string>
              </property>
             </column>
             <column>
              <property name="text">
               <string>New Column</string>
              </property>
             </column>
             <column>
              <property name="text">
               <string>New Column</string>
              </property>
             </column>
             <column>
              <property name="text">
               <string>New Column</string>
              </property>
             </column>
            </widget>
           </item>
          </layout>
         </widget>
         <widget class="QWidget" name="tab_4">
          <attribute name="title">
           <string>ÏòàÏïΩ</string>
          </attribute>
         </widget>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QFrame" name="frame_center">
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <layout class="QGridLayout" name="gridLayout_3">
       <item row="4" column="0">
        <widget class="QLabel" name="label">
         <property name="text">
          <string>Ï≤¥Í≤∞ÌòÑÌô©</string>
         </property>
        </widget>
       </item>
       <item row="2" column="0" rowspan="2" colspan="5">
        <widget class="QTableWidget" name="table_stocklist">
         <column>
          <property name="text">
           <string>Ï¢ÖÎ™©Î™Ö</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>ÏΩîÎìú</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>ÌòÑÏû¨Í∞Ä</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>ÎåÄÎπÑÏú®</string>
          </property>
         </column>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QTableWidget" name="tableWidget_2">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <column>
          <property name="text">
           <string>New Column</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>New Column</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>New Column</string>
          </property>
         </column>
        </widget>
       </item>
       <item row="0" column="0">
        <widget class="QTableWidget" name="tableWidget_4">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <column>
          <property name="text">
           <string>New Column</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>New Column</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>New Column</string>
          </property>
         </column>
        </widget>
       </item>
       <item row="5" column="0" colspan="5">
        <widget class="QTableWidget" name="table_trades">
         <column>
          <property name="text">
           <string>ÏãúÍ∞Ñ</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Í∞ÄÍ≤©</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>ÏàòÎüâ</string>
          </property>
         </column>
        </widget>
       </item>
       <item row="1" column="1" colspan="4">
        <widget class="QTableWidget" name="tableWidget_3">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <column>
          <property name="text">
           <string>New Column</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>New Column</string>
          </property>
         </column>
        </widget>
       </item>
       <item row="0" column="1" colspan="4">
        <widget class="QTreeWidget" name="treeWidget_3">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <column>
          <property name="text">
           <string>New Column</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>New Column</string>
          </property>
         </column>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QFrame" name="frame_right">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Fixed" vsizetype="Ignored">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_right">
       <item>
        <widget class="QLabel" name="label_chart">
         <property name="text">
          <string>üìà Ï∞®Ìä∏ ÏòÅÏó≠ (ÎØ∏Íµ¨ÌòÑ)</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>0</width>
           <height>0</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QPushButton" name="button_refresh">
         <property name="text">
          <string>ÏÉàÎ°úÍ≥†Ïπ®</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QDockWidget" name="dockWidget">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="autoFillBackground">
    <bool>true</bool>
   </property>
   <property name="floating">
    <bool>true</bool>
   </property>
   <attribute name="dockWidgetArea">
    <number>2</number>
   </attribute>
   <widget class="QWidget" name="dockWidgetContents">
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QPushButton" name="pushButton_3">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Expanding">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>ÏõêÍ≤©ÏßÄÏõê</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Expanding">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>ÌÜµÌï©Ï£ºÎ¨∏</string>
       </property>
       <property name="iconSize">
        <size>
         <width>16</width>
         <height>30</height>
        </size>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_9">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Expanding">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>ÏûîÍ≥†Î™©Î°ù</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_8">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Expanding">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>Í±∞ÎûòÎÇ¥Ïó≠</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_7">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Expanding">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>Ïò§Î≤ÑÏã†Ï≤≠</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_6">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Expanding">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>Î™®ÏùòÏã†Ï≤≠</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_5">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Expanding">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>ÌôòÍ≤ΩÏÑ§Ï†ï</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_4">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Expanding">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>1:1Î¨∏Ïùò</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_2">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Expanding">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>Í≥µÏßÄÏÇ¨Ìï≠</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
